packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -1071,7 +1071,7 @@ ensureIndex(Posts,
 Posts.addView("nominations2019", (terms: PostsViewTerms) => {
   return {
     selector: {
-      nominationCount2019: { $gt: 0 }
+      nominationCount2019: { $gte: 2 }
```

Comment:
This also hides posts with one nomination from `/nominations`, which I don't think was intended.

Reply:
I was aware of it and decided it was a fine sacrifice to get it out of the door, but agree that it's a bug and should be marked as such.